modules/apps/object/object-web/src/main/resources/META-INF/resources/list_type_definitions/edit_list_type_definition.jsp

```diff
@@ -143,11 +143,15 @@ ViewListTypeEntriesDisplayContext viewListTypeEntriesDisplayContext = (ViewListT
 							'<%= LanguageUtil.get(request, "the-picklist-was-updated-successfully") %>',
 						type: 'success',
 					});
+					const parentWindow = Liferay.Util.getOpener();
 
 					setTimeout(() => {
-						const parentWindow = Liferay.Util.getOpener();
 						parentWindow.Liferay.fire('close-side-panel');
 					}, 1500);
+
+					setTimeout(() => {
+						parentWindow.location.reload();
+					}, 2000);
```

Comment:
Use a single `setTimeout()`:
```tsx
					setTimeout(() => {
						const parentWindow = Liferay.Util.getOpener();
						parentWindow.Liferay.fire('close-side-panel');
						parentWindow.location.reload()
					}, 1500);
```

Reply:
Hi @miltonmc! Unfortunately it doesn't work. :confused:
I've tried to do this before splitting into two setTimeout().
By doing the change the save doesn't work and a console error appears which I didn't undestand:
_com_liferay_object_web_internal_list_type_portlet_portlet_ListTypeDefinitionsPortlet_saveListTypeDefinition is not defined
    at HTMLFormElement.onsubmit `

Reply:
I think it's better to show you on Monday.